### PR TITLE
Views for the blog

### PR DESCRIPTION
**Pull request by osamaashraf6**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

> In this project, The Blog app will be a classic example of a blog website. I will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.🪄🎈

- [ ] Implement the design from the sneak peek wireframes.
- [ ] Implement the views for the blog

## Description of the task to be completed 👎 

- [ ] Run a program using the command line.👌
- [ ] Implement the design from the sneak peek wireframes.
- [ ] Make sure that for each action you check:..💯
- [ ] Initialize my project with Git.💯

  
## How should it be tested

Clone the repository
```
git clone https://github.com/osamaashraf6/Blog-App
```
Navigate to the folder
```
cd Blog-App
```

Checkout the branch
```
git checkout views
```

#### Thanks for reviewing my PR 😊
Don't forget to give a ⭐️ to support me 😃

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+</div>
 
 <h2>Posts:</h2>
 
-<ul>
+<ul class="useridposts">
   <% @user.posts.each do |post| %>
-    <li><%= link_to post.title, user_post_path(@user, post) %></li>
+   <div class="container">
+    <li>
```
Comment (elisha2kyakpo1) [inline app/views/users/show.html.erb]: - [ ] Here we do not need all the posts, fetching all the posts should be done using the button `see all posts`. Kindly make use of the `recent_posts` callback function you created in the user model to render the most recent 3 posts.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+    </div></div>
+</div>
 
-<ul>
-  <% @posts.each do |post| %>
-    <li><%= link_to post.title, user_post_path(@user, post) %></li>
+<ul class="useridposts">
+  <% @user.posts.each do |post| %>
+<div class="container">
+    <li class="postshow">
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb]: - [ ] Let's use our `recent_five_comments` callback function to render the 5 most recent comments for a post. We do not need to hardcode the comments. You can use the rails console to test if this function is working. Create more than 5 comments on a post, and run the server to see if that post will only show 5 comments in the browser.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,7 +1,37 @@
...
+    <span> likes: X <%=  post.commentsCounter  %> </span>
+    <span> comments: Y <%=  post.likesCounter %> </span>
+
+
+<div class="postcomment">
+<p>username: comment 1</p>
+<p>username: comment 2</p>
+<p>username: comment 3</p>
+<p>username: comment 4</p>
+</div>
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb]: - [ ] [OPTIONAL] We do not need these hardcoded comments

File: app/views/posts/show.html.erb [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
+<div class="flex"><span><%= @post.id %># By : <%= @user.name %>  </span> <span> likes: X <%=  @post.commentsCounter  %> </span> <span> comments: Y <%=  @post.likesCounter %> </span></div>
+<p><%= @post.text %></p>
 
+</div>
+<div class="showpostcomment">
+<p>username: comment 1</p>
+<p>username: comment 2</p>
+<p>username: comment 3</p>
+<p>username: comment 4</p>
+</div>
```
Comment (elisha2kyakpo1) [inline app/views/posts/show.html.erb]: - [ ] [OPTIONAL] There is no need to do this because the comments should come from the database.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @osamaashraf6, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look excellent and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Database column names should be camel-cased, `postsCounter` should be `posts_counter`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @osamaashraf6 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- You have good commit messages. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**